fyndiq/fyndiq-ui
# Jest + Enzyme testing

## Overview

Add [Jest](http://facebook.github.io/jest/) testing suite to test individual components.

## How to test

- `npm i`
- `npm run jest`

The tests should pass. Try to change one component (remove [this line](https://github.com/fyndiq/fyndiq-ui/blob/92a2ea5b124a2a2e25fa7b81fa9d1c9569ddd7d3/packages/fyndiq-component-button/src/index.js#L15) for instance) and see how it affects the tests.
